app/src/main/java/org/puder/trs80/MainActivity.java

```diff
@@ -417,7 +417,7 @@ private void showCommunity() {
 
     private void downloadROMs() {
         AlertDialog.Builder builder = AlertDialogUtil.createAlertDialog(this,
-                R.string.title_initial_setup, R.drawable.warning_icon, R.string.initial_setup);
+                R.string.title_initial_setup, R.drawable.ic_info_black_24px, R.string.initial_setup);
```

Comment:
This was the name as it was downloaded from material design web site. I don't care what it's called.  Would you prefer simply `ic_info_black` ?